docs/operation/commands-shortcuts.md

```diff
@@ -40,6 +40,26 @@ Below are the available commands in the current Hummingbot release.
 | `stop`            | Stop gateway docker instance                                  |
 | `test-connection` | Ping gateway api server                                       |
 
+Users can also use `gateway [command] -h` to see individual uses of each command.
+
```

Comment:
Add a section here to describe `gateway -h` and print its output, which shows all the gateway commands

Reply:
This has been resolved in the latest commit